### DISCUSSION TITLE
DIsable splash screen?

### DISCUSSION THREAD

**Discussion by robotshateme** [chocolatey/ChocolateyGUI#872] [Q&A]
Is there currently a way to disable the splash screen on startup?

**Comment by gep13** [comment]
@robotshateme no, there currently isn't anyway to do this.  Can I ask why you would want/need this as an option?  Thanks

**Reply by atesin** [reply]
hi @robotshateme ... i feel your response a little harsh, i also feel affected so i will answer in the name of users

the chocolateygui splash screen is very annoying when we are also using our pc for other tasks and the splash places in front, ocuppying a large screen area that cannot be backgrounded, moved, closed or minimized; effectively blocking our pc for being used for about 30 seconds... feeling forced to wait, wasting our time and patience, and growing up an unfair hate for chocolateygui

we know chocolateygui is cool, that is for we installed it... so there is no need to remark it everytime with a monstrous and stubborn splash screen... an option to disable it would be excelent and appreciated

see issue #890 also
